src/LoginFormContainer.test.jsx

```diff
@@ -0,0 +1,75 @@
+import { render, fireEvent } from '@testing-library/react';
+import { useDispatch, useSelector } from 'react-redux';
+import LoginFormContainer from './LoginFormContainer';
+
+jest.mock('react-redux');
+
+describe('LoginFormContainer', () => {
+  const dispatch = jest.fn();
+
+  beforeEach(() => {
+    dispatch.mockClear();
+    useDispatch.mockImplementation(() => dispatch);
+    useSelector.mockImplementation((selector) => selector({
+      loginFields: {
+        email: 'test@test',
+        password: '1234',
+      },
+      accessToken: given.accessToken,
+    }));
+  });
```

Comment:
mock을 초기화 하는 것은 다른 일을 하는 것이니 구분해서 작성해주면 좋을 것 같아요. 그리고 모든 mock을 초기화하려면 다음과 같이 할 수 있습니다. 혹은 jest 설정을 변경하는 방법도 있는데, 한 번 적용해 보세요.

```suggestion
  beforeEach(() => {
    jest.clearAllMocks();
  });

  beforeEach(() => {
    useDispatch.mockImplementation(() => dispatch);
    useSelector.mockImplementation((selector) => selector({
      loginFields: {
        email: 'test@test',
        password: '1234',
      },
      accessToken: given.accessToken,
    }));
  });
```

See also
  - https://jestjs.io/docs/configuration#clearmocks-boolean